backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -174,7 +175,11 @@ export class AdminResolver {
     if (!lastUserTransaction) {
       newBalance = 0
     } else {
-      newBalance = lastUserTransaction.balance
+      newBalance = await calculateDecay(
+        lastUserTransaction.balance,
+        lastUserTransaction.balanceDate,
+        new Date(),
```

Comment:
Should this date be the same as in line 204?

Reply:
Change it store ``new Date()`` in a ``variable`` and use it everywhere where we used ``new Date()``